packages/client/hmi-client/src/services/model.ts

```diff
@@ -182,3 +182,27 @@ export function isInitial(obj: Initial | ModelParameter | null): obj is Initial
 export function isModelParameter(obj: Initial | ModelParameter | null): obj is ModelParameter {
 	return obj !== null && 'id' in obj;
 }
+
+export function stringToLatex(expression: string): string {
+	// First, escape backslashes to avoid breaking LaTeX syntax
+	let latexExpression = expression.replace(/\\/g, '\\\\');
+
+	// Wrap everything after the first underscore in {} for each variable
+	// and add a \ before subsequent underscores
+	latexExpression = expression.replace(/(_)([a-zA-Z0-9_]+)/g, (_match, p1, p2) => {
+		// Replace subsequent underscores in p2 with \_
+		const modifiedP2 = p2.replace(/_/g, '\\_');
+		return `${p1}{${modifiedP2}}`;
+	});
+
+	// (Unsure about this) Convert * to space (implicit multiplication) for LaTeX
+	// latexExpression = latexExpression.replace(/\*/g, ' ');
+
+	// Convert ^ to LaTeX superscript notation
+	latexExpression = latexExpression.replace(/\^([a-zA-Z0-9]+)/g, '^{$1}');
+
+	// Detect and convert fractions a/b to \frac{a}{b}
+	latexExpression = latexExpression.replace(/([a-zA-Z0-9]+)\/([a-zA-Z0-9]+)/g, '\\frac{$1}{$2}');
+
+	return latexExpression;
+}
```

Comment:
This is the sort of function that would love a few unit test cases, not only to test but to demonstrate functionality. 

For example it is a bit unusual to see that we need superscript `^` conversion here ... what is the use-case?

Reply:
Assuming exponents a written as `x^2` in the string format, I'd have to convert it to `x^{2}` for latex